packages/model-viewer/src/features/scene-graph/api.ts

```diff
@@ -228,11 +228,23 @@ export declare interface Image {
    */
   readonly uri?: string;
 
+  /**
+   * The bufferView of the image, if it is embedded.
+   */
+  readonly bufferView?: number
```

Comment:
This is so I can make a cache identifier; I should either have a uri or a bufferView, and either way it should be unique per image.